src/secp256k1.c

```diff
@@ -57,6 +57,8 @@
     } \
 } while(0)
 
+/* Note that whenever you change the context struct, you must also change the
+ * context_eq function. */
```

Comment:
maybe add "in the tests"

Reply:
My thinking was that if we ever move this then it's unlikely we update this line. So just having someone searching it is more robust.

Reply:
Agreed.